### ISSUE TITLE
Make sure to note (technically) breaking change in "viewingHint" : "continuous" in changelog for 2.1

### ISSUE THREAD

**Issue by jpstroop** [IIIF/iiif.io#486]
See conversation around https://github.com/IIIF/iiif.io/issues/451#issuecomment-99571642

**Comment by azaroth42** [comment]
Blocked by #497 as we need the clarification and changes note to create this note in.
:point_right: :point_right:  @jpstroop @zimeon @mikeapp @tomcrane

**Comment by jpstroop** [comment]
My comment on the PR is my only concern w/ this, but that can be turned into a separate issue if others agree. Otherwise I'm OK w/ merging #497.

**Comment by azaroth42** [comment]
Updated:

http://470_semver.iiif.io/api/annex/notes/scheduled-changes.html

R

On Fri, Jul 17, 2015 at 6:34 AM, Jon Stroop <notifications@github.com>
wrote:

> My comment on the PR is my only concern w/ this, but that can be turned
> into a separate issue if others agree. Otherwise I'm OK w/ merging #497
> <https://github.com/IIIF/iiif.io/pull/497>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/IIIF/iiif.io/issues/486#issuecomment-122275779>.
>



-- 
Rob Sanderson
Information Standards Advocate
Digital Library Systems and Services
Stanford, CA 94305

**Comment by azaroth42** [comment]
Done
